components/arm/client.go

```diff
@@ -8,31 +8,36 @@ import (
 	"github.com/edaniels/golog"
 	commonpb "go.viam.com/api/common/v1"
 	pb "go.viam.com/api/component/arm/v1"
+	robotpb "go.viam.com/api/robot/v1"
 	"go.viam.com/utils/protoutils"
 	"go.viam.com/utils/rpc"
 
 	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/config"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/spatialmath"
 )
 
-var errArmClientInputsNotSupport = errors.New("arm client does not support inputs directly")
+var errArmClientModelNotValid = errors.New("arm client has incorrectly configured model and could not complete the operation")
```

Comment:
This is probably better worded as "unable to retrieve a valid arm frame from remote robot" or something of that sort.

We don't want to be specific about the reason for the error (incorrectly configured) unless we're sure that's what it is and not some other issue (packet loss during init for example)

Reply:
changed to `errors.New("unable to retrieve a valid arm model from arm client")`